bpfman/src/lib.rs

```diff
@@ -215,6 +224,10 @@ pub async fn add_program(mut program: Program) -> Result<Program, BpfmanError> {
 
             add_multi_attach_program(root_db, &mut program, &mut image_manager, config).await
         }
+        Program::Tcx(_) => {
+            program.set_if_index(get_ifindex(&program.if_name().unwrap())?)?;
```

Comment:
Can this `unwrap()` fail? Would prefer not to have it if possible.

Reply:
Done.